src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        truck.setName("t1");
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
It's better to use the parent class (or interface) type for variable declaration when possible. This encourages programming to an interface and can make your program more flexible. Here, you can declare 'bulldozer' as 'Machine' type.